tests/python/pants_test/engine/legacy/test_parser.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import unittest
+
+from pants.build_graph.build_file_aliases import BuildFileAliases
+from pants.engine.legacy.parser import LegacyPythonCallbacksParser
+from pants.engine.parser import EmptyTable
+
+
+class LegacyPythonCallbacksParserTest(unittest.TestCase):
+
+  def test_no_import_sideeffects(self):
+    # A parser with no symbols registered.
+    parser = LegacyPythonCallbacksParser(EmptyTable(), BuildFileAliases())
+    # Call to import a module should succeed.
+    parser.parse('/dev/null', '''import os; os.path.join('x', 'y')''')
```

Comment:
Maybe just "import ... ')" here and below, '''import...')''' - lots of ticks!

Reply:
My take is that consistency of string styles is more important than the total length... then again, using single quotes is only what I'm currently remembering as our convention, and I haven't taken a survey of quoting styles, heh. Will leave for now.

Reply:
Yeah, this is ill-defined. I know I personally always use `"""` when tripling - due to doc comments setting that precedent, and I stick to `'` when I can, then using `"` if I can't before passing to `"""` for multiline.